packages/bpk-docs/src/pages/BannerAlertsPage/BannerAlertsPage.js

```diff
@@ -84,6 +84,81 @@ BpkBannerDismissable.defaultProps = {
   type: null,
 };
 
+// eslint-disable-next-line react/no-multi-comp
+class BpkBannerAlertDismissDemo extends Component {
+  constructor() {
+    super();
+
+    this.newBannerAlerts = [
```

Comment:
Why the `new` part?

Reply:
I guess what I'm getting at is it reads a bit funny - you can probably just call it `bannerAlerts`